pandas/parser.pyx

```diff
@@ -483,10 +484,13 @@ cdef class TextReader:
         self.verbose = verbose
         self.low_memory = low_memory
         self.parser.converter = xstrtod
+        self.parser.converter_nogil = True
```

Comment:
while we are at it, can you rename ``self.parser.converter`` -> ``self.parser.double_converter`` (and same with ``convert_nogil`` -> ``double_converter_nogil``